agent/Makefile

```diff
@@ -1,16 +1,21 @@
 .DEFAULT_GOAL := build
 SHELL := bash
 
-export VERSION:=$(shell cat ../VERSION)
+export VERSION:=$(shell ../version.sh)
+export VERSION_TAG:=$(shell ../version.sh -t)
+export VERSION_DOCKER := $(shell ../version.sh -d)
+
 export GO111MODULE := on
 export DOCKER_REPO ?= determinedai
 
 FULL_COMMIT = $(shell git rev-parse HEAD)
 SHORT_COMMIT = $(shell git rev-parse HEAD | head -c9)
 PROJECT_NAME = determined-agent
+PROJECT_NAME_DRYRUN = determined-agent-dryrun
 EE_PROJECT_NAME = hpe-mlde-agent
 ARCHS = amd64 arm64
 MULTI_ARCH_IMAGES = $(shell for arch in $(ARCHS); do echo $(DOCKER_REPO)/$(PROJECT_NAME):$(FULL_COMMIT)-$$arch; done)
+MULTI_ARCH_IMAGES_DRYRUN = $(shell for arch in $(ARCHS); do echo $(DOCKER_REPO)$(PROJECT_NAME_DRYRUN):$(FULL_COMMIT)-$$arch; done)
```

Comment:
this type of thing looks like it's gonna be a pain to maintain. my gut reaction here is that i would expect `PROJECT_NAME` to simply change if dryrun. this all looks kinda delicate and im hoping dryrun includes more behavior logic than config changes, so that way this comment won't be justified.

Reply:
I might also be biased towards complexity